pyanaconda/modules/storage/checker/checker.py

```diff
@@ -34,3 +35,14 @@ class StorageCheckerModule(KickstartBaseModule):
     def publish(self):
         """Publish the module."""
         DBus.publish_object(STORAGE_CHECKER.object_path, StorageCheckerInterface(self))
+
+    def set_constraint(self, name, value):
+        """Set a constraint to a new value.
+
+        KeyError will be raised if the constraint does not exist.
```

Comment:
There is ``:raise:`` for this kind of documentation.

Reply:
Fixed.